mesonbuild/compilers.py

```diff
@@ -74,38 +77,50 @@ def is_library(fname):
                                           'debug' : [],
                                           'debugoptimized' : [],
                                           'release' : [],
+                                          'minsize' : [],
                                          })
 else:
     gnulike_buildtype_linker_args.update({'plain' : [],
                                           'debug' : [],
                                           'debugoptimized' : [],
                                           'release' : ['-Wl,-O1'],
+                                          'minsize' : [],
                                          })
 
 msvc_buildtype_linker_args = {'plain' : [],
                               'debug' : [],
                               'debugoptimized' : [],
-                              'release' : []}
+                              'release' : [],
+                              'minsize' : [],
```

Comment:
Should probably add `/INCREMENTAL:NO` here since that reduces code size by not leaving padding or thunks.

https://msdn.microsoft.com/en-us/library/4khtbfyf.aspx